.jpb/jpb-settings.xml

```diff
@@ -3,7 +3,7 @@
   <component name="DatabaseMigrationSettings" lastSelectedDirectory="src/main/resources/db/migration" />
   <component name="PersistenceUnitSettings">
     <persistence-units>
-      <persistence-unit name="Default">
+      <persistence-unit moduleName="mmotodo" name="Default">
```

Comment:
Is this a change that will stay like this or what is moduleName? Is this even a file that needs to be in the repo or something that flyway might recreate when loaded into intellij?

Reply:
I don't know what that is. It was included by the guy who worked with the database branch